tracking/translations_parser/parser.py

```diff
@@ -322,8 +326,12 @@ def parse_marian_context(self, logs_iter: Iterator[tuple[list[tuple[str]], str]]
         headers: list[tuple[str]] = []
         # Consume first lines until we get the Marian header
         while ("marian",) not in headers:
-            headers, text = next(logs_iter)
-            logger.debug(f"Marian header not found in: headers={headers} text={text.strip()}")
+            try:
+                headers, text = next(logs_iter)
+                logger.debug(f"Marian header not found in: headers={headers} text={text.strip()}")
+            except StopIteration as exception:
+                logger.debug("Could not find a [marian] entry in the training log.")
+                raise exception
```

Comment:
maybe worth raising something else instead of StopIteration that can be confusing. Like `raise ValueError("Could not find a [marian] entry in the training log.")`